app/views/static.py

```diff
@@ -1,25 +1,38 @@
-from flask import Blueprint
+from flask import Blueprint, current_app
 from flask_themes2 import render_theme_template
 
-from app.helpers import template_helper
-
+from app.globals import get_session_store
+from app.utilities.schema import load_schema_from_params
 
 from structlog import get_logger
 
 logger = get_logger()
 
-contact_blueprint = Blueprint('contact', __name__,)
+contact_blueprint = Blueprint(name='contact', import_name=__name__)
 
 
 @contact_blueprint.route('/contact-us', methods=['GET'])
 def contact():
-    return render_theme_template('default', 'static/contact-us.html')
+    session = None
+    survey_id = None
+    session_store = get_session_store()
 
+    if session_store:
+        session = session_store.session_data
+        schema = load_schema_from_params(session.eq_id, session.form_type)
```

Comment:
This will fail with surveys launched from URL. Once merged i'll fix it as part of #1450.